src/cli/Command.php

```diff
@@ -162,9 +162,11 @@ public function actionExec($id, $ttr, $attempt, $pid)
      */
     protected function handleMessage($id, $message, $ttr, $attempt)
     {
-        // Child process command: php yii queue/exec "id" "ttr" "attempt" "pid"
+        // Child process command: php -c <PATH_TO_INI_FILE> yii queue/exec "id" "ttr" "attempt" "pid"
         $cmd = [
             $this->phpBinary,
+            '-c',
+            get_cfg_var('cfg_file_path'),
```

Comment:
It may return `false` in case php.ini isn't used. In this case `$cmd` should not contain `-c`.

Reply:
I tried it without passing `-c` argument and it just returns the default php.ini file like /etc/php/7.x/cli/php.ini. But there is no problem to add `if`, if you believe the `false` is possible, I'll do it. Also I got about CHANGELOG.